src/server/ProxiedApp.ts

```diff
@@ -94,6 +94,10 @@ export class ProxiedApp implements IApp {
             logger.error(e);
             logger.debug(`'${method}' was unsuccessful.`);
 
+            if (e instanceof Toast.Error) {
```

Comment:
Ideally, this case would use the same inheritance of `AppsEngineException` as seen below, since that "pattern" is used in some other places as well. Is this use case different in a way that prevents it from using that?